default.env

```
@@ -15,3 +15,6 @@ REACT_APP_CONF_API_URL=http://localtest.me:5000
 # variable that drives whether to render MME graph using mock data
 # set to true if so
 USE_DEMO=false
+
+#url for patient search
+FEMR_BASE_DOMAIN:http://localtest.me:5000
```

`FEMR_BASE_DOMAIN` is what `REACT_APP_DASHBOARD_URL` is based on, but you should use `REACT_APP_DASHBOARD_URL`, which should be available at `/env.json` of the frontend

```suggestion
REACT_APP_DASHBOARD_URL:http://localtest.me:5000
```